scripts/error-codes/codes.json

```diff
@@ -144,57 +144,58 @@
   "142": "Expected onBeforeMountComponent() parent and onSetChildren() to be consistent (%s has parents %s and %s).",
   "143": "React.Children.only expected to receive a single React element child.",
   "144": "React.PropTypes type checking code is stripped in production.",
```

Comment:
Seems like we need to change `144` to be `"Item must have been set",` and then it'll be good on next generation